packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Cover.Example.tsx

```diff
@@ -48,31 +55,34 @@ export class CalloutCoverExample extends React.Component<{}, ICalloutCoverExampl
     // Large beak will disable some position to avoid beak over the callout edge.
     return (
       <div className="ms-CalloutExample">
-        <div className="ms-CalloutExample-configArea">
+        <div className={css(classNames.msCalloutExampleConfigArea)}>
```

Comment:
If there's just one class name, you don't need to use the css helper (same comment applies to all example files)

Reply:
Fixed!